R/dist-fitting.R

```diff
@@ -207,7 +221,7 @@ fit_info <- function (data, distribution) {
   } else  {
     params <- tried_fit$estimate
   }
-  list(params = params, distribution = distribution, chisq = chi_sq(list(data = data, distribution = distribution, estimate = params)))
+  list(params = params, distribution = convert_dist(distribution), chisq = chi_sq(list(data = data, distribution = convert_dist(distribution), estimate = params)))
```

Comment:
Same answer. If anything, it's a safety measure and makes testing easier. If the distribution is already converted, it stays converted: `convert_dist("normal")=convert_dist("norm") ="norm"